### ISSUE TITLE
`Tuple` is incompatible with `tuple`

### ISSUE THREAD

**Issue by decorator-factory** [microsoft/pyright#1120]
Python 3.9 and latest version of Pyright/Pylance.

```py
def f(xs: Tuple[str, ...]):
    return xs

f(tuple(["a", "b"]))
```

error:
```
"Argument of type "tuple[str, ...]" cannot be assigned to parameter "xs" of type "Tuple[str, ...]" in function "f"
  "tuple[str, ...]" is incompatible with "Tuple[str, ...]"
```

**Comment by erictraut** [comment]
I'm not able to repro this with either Pylance or Pyright. Can you double-check your code sample above and verify that it repros the problem for you? Are there any other special settings that you're using (e.g. a different version of typeshed stubs than those that ship with Pylance/Pyright)?

**Comment by decorator-factory** [comment]
This is some kind of a heisenbug. Sometimes it appears, and sometimes it does not (i.e. when I open the module for the second time). I don't have anything special settings. I'll investigate this more.

**Comment by erictraut** [comment]
I was finally able to repro this. A fix will be in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.83, which was just published. It will also be included in the next release of pylance.
